scripts/gaia/README.md

```diff
@@ -0,0 +1,40 @@
+# Gaia DR3
+
+This folder contains a dataset based on the Gaia DR3 catalog. The dataset is a subset of the full catalog, containing only stars which have BP/RP low-resolution spectra available. For this dataset, we prepare astrometry, photometry, photometrically-estimated parameters, and low-resolution spectra in the form of spectral coefficients.
+
+For more information on Gaia data, see https://gea.esac.esa.int/archive/documentation/GDR3/Gaia_archive/chap_datamodel/sec_dm_main_source_catalogue/ssec_dm_gaia_source.html.
+
+## Astrometry
+
+This includes:
+
+- `ra` and `dec` in degrees
+- `parallax` in mas
+- `pmra` and `pmdec` in mas/yr
+- `ra_error`, `dec_error`, `parallax_error`, `pmra_error`, and `pmdec_error` in mas
+- `ra_dec_corr`, `ra_parallax_corr`, `ra_pmra_corr`, `ra_pmdec_corr`, `dec_parallax_corr`, `dec_pmra_corr`, `dec_pmdec_corr`, `parallax_pmra_corr`, `parallax_pmdec_corr`, and `pmra_pmdec_corr` as correlation coefficients
+
```

Comment:
Let's add a short explanation on RV here - I can see it's added in the dataset 😄

Reply:
README expanded